tests/test_content.py

```diff
@@ -421,6 +421,83 @@ def test_datetime():
     assert new_result == '2001-02-03 04:05'  # output
 
 
+def test_nested_align():
+    """
+    # Nested Formats
```

Comment:
@zerok. I am not sure that I understand the request. Do you want me to remove the hash? It seems consistent with all the other headings.